html/registrationuser.html

```diff
@@ -0,0 +1,205 @@
+              event.preventDefault();
+              alert("Enrollment Number Already Exists");
+            } else {
+              $("#enroll_result").empty();
+              checkvariable[0]=1;
+            }
+          }
+        });
+      });
+
+        $("#username").blur(function(event) {
+        var input = "username";
+        var inputvalue = $("#username").val();
+        let tablename = "student";
+
+        $.ajax({
+          type: "post",
+          url: "../php/check.php",
+          data: { input: input, inputvalue: inputvalue, tablename: tablename },
+          dataType: "json",
+          success: function(response) {
+            if (!response) {
+              $("#username_result").html("Username Already Exists");
+              event.preventDefault();
+              alert("username Already Exists");
+            } else {
+              $("#username_result").empty();
+              checkvariable[1]=1;
+            }
+          }
+        });
+        });
+
+      var check_password = 0;
+      function pass_check() {
+        var p = document.getElementById("p").value;
+        var p2 = document.getElementById("p2").value;
+        if (p2) {
+          if (p != p2) {
+            var msg = "Passwords don't match".fontcolor("red");
+            document.getElementById("pass").innerHTML = msg;
+            check_password = 0;
+          } else {
+            var msg = "Passwords match".fontcolor("green");
+            document.getElementById("pass").innerHTML = msg;
+            check_password = 1;
+            checkvariable[2]=1;
+          }
+        }
+      }
+
+      $("#email").blur(function(event) {
+        var input = "email";
+        var inputvalue = $("#email").val();
+        var tablename = "student";
+
+        $.ajax({
+          type: "post",
+          url: "../php/check.php",
+          data: { input: input, inputvalue: inputvalue, tablename: tablename },
+          dataType: "json",
+          success: function(response) {
+            if (!response) {
+              $("#email_result").html("Email Already Exists");
+              event.preventDefault();
+              alert("Email Already Exists");
+            } else {
+              $("#email_result").empty();
+              checkvariable[3]=1;
+            }
+          }
+        });
+      });
+
+      $('#btnSubmit').on('click', function(){
+ 	var name = $('#name').val();
+  var username = $('#username').val();
+ 	var email = $('#email').val();
+ 	var password = $('#password').val();
+  var enroll = $('#enroll').val();
+  var bhawan = $('#bhawan').val();
+  var room = $('#room').val();
+  var mobile = $('#mobile').val();
+ 	if (checkvariable[0]==0 || checkvariable[1]==0) {
+	  $('#error').text('Fix the errors in the form first');
+	}
+  else if (checkvariable[2]==0 || checkvariable[3]==0) {
+    $('#error').text('Fix the errors in the form first');
+  }
+  else{
+      // proceed with form submission
+      $.ajax({
```

Comment:
instead of submitting the request to php, the php file itself is getting downloaded, check that once.